tests/e2e/commerce/agreement/conftest.py

```diff
@@ -0,0 +1,64 @@
+import datetime as dt
+
+import pytest
+from dateutil.relativedelta import relativedelta
+from freezegun import freeze_time
+
+
+def get_iso_dt_str(dt_obj: dt.datetime) -> str:
```

Comment:
We are not using typing in the test folder ->

<img width="423" height="594" alt="Image" src="https://github.com/user-attachments/assets/390d8cf5-844a-4f44-bfe5-eb96cacabdc0" />

So it should be removed from here and from the rest of the test files.